lib/scrabble/tilebag.rb

```diff
@@ -0,0 +1,86 @@
+module Scrabble
+  class TileBag
+    attr_reader :tiles
+
+
+    ## -------------------------------------------------------------------------
+    # CLASS METHODS
+
+
+    def initialize
+      create_tiles
+      shuffle_tiles # yes, I do call this inside create_tiles
```

Comment:
I prefer shuffling multiple times, because it feels more random and less... like shifting every other card three cards down. As for shuffling only at the beginning of a game or pile of things, I think that would be necessary for games that have implied orders: like drawing cards from a deck. But when I think of a tile bag, I think of the letters shifting around every time I pick up and set the bag down again. So I want to shuffle every time.